client/src/components/body/lab2/reading.js

```diff
@@ -0,0 +1,151 @@
+                  pink, red, black, white, grey, and turquoise. This CVD is extremely
+                  rare.
+                  </p>
+          </li>
+          <li><h3>Monochromacy (mon-o-chro-ma-cy):</h3>
+                  <p className="center">
+                  Individuals with this deficiency can see no colors at all. Everything
+                  is a shade of grey that ranges between white and black. This is
+                  incredibly rare (approximately 1 in 33,000 people are diagnosed With
+                  this condition). Due to the range of colors these individuals can see,
+                  it can make some tasks in life difficult to preform.
+                  </p>
+          </li>
+        </ul>
+        
+        <h3 className="center">
+          Each form of CVD has varying levels of severity.
+        </h3>
+        <div>
+          <h3 className="center">
+            What, in the eye, is causing colors to be perceived in this way?
+          </h3>
+          <p className="center">
+            For individuals with Protanomaly, Deuteranomaly, and Tritanomaly, they
+            are considered dichromatic, meaning that they only have two types of
+            cones in their eyes to perceived colors. People without these
+            deficiencies have three cones and each cone is responsible for
+            perceiving red, green, and blue. This is where the reduced sensitivity
+            comes from, as the reduction in one cone causes a section of the color
+            spectrum to not be viewable to the individual. This section happens to
+            overlap for individuals with Protanomaly and Deuteranomaly, which is
+            why some individuals may refer to their deficiency as 'red/green
+            colorblindness', as these individuals may tend to see the world in
+            very similar ways.
+          </p>
+        </div>
+        <div>
+          <h3 className="center">
+            Causes of Color Vision Deficiencies:
+          </h3>
+          <p className="center">
+            Color Vision Deficiencies are normally caused by genetic conditions
+            and are usually inheritied from an individual's parents. Men are more
+            often impacted than women because the gene is carried in the X
+            chromosome. There can be some other cases that cause color vision
+            deficiencies however their causes are still being researched to
+            determine the exact cause.
+          </p>
+        </div>
+        <h3 className="center">
+          At this moment in time, there is no cure to these deficiencies.
+        </h3>
+        <h3 className="center">
+          For more information, please visit the following websites:
+        </h3>
+        <p className="center">
+  
+            <a href="http://www.colourblindawareness.org/" target="_blank" rel="noopener noreferrer" className="padding">
+            colourblindawareness.org
```

Comment:
This link and the next link appear directly side by side, I think maybe they should be listed in a vertical list?